README.md

```diff
@@ -97,14 +97,13 @@ this suite:
 
 ### Selenium Tests
 
-this suite has not been implemented yet.
+run these with `pytest --base-url http://localhost:3000 --driver Chrome pytest-selenium/tests`
```

Comment:
```suggestion
make venv
source ./.venv/bin/activate
run these with `pytest --base-url http://localhost:3000 --driver Chrome pytest-selenium/tests`
```

This _partially_ worked for me but I got the following error (do we need separate instructions for installing chromedriver?):

```
selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH.
```

Maybe it would make sense to add a Makefile target instead? Like `make test-local` or something?